site/docs/references/formatters.md

```diff
@@ -126,9 +130,3 @@ You can use nested formatters:
         return Math.PI * multiplyBy;
```

Comment:
replace this with the following example, and replace usage in the rest of the example:

```js
/* ./sin.js */
module.exports = function(degree) {
    return Math.sin(Number(degree));
}